### ISSUE TITLE
Error with charset meta tag

### ISSUE THREAD

**Issue by pensierinmusica** [htmllint/htmllint#156]
Hi,

The linter is returning an error for:

```html
<html>

  <head>
    <meta charset="utf-8">
  </head>

  <body></body>

</html>
```

Reporting: `(E042) tag is not closed` (I'm using the default options).

Void elements though should not be closed according to the [HTML5 standard](http://stackoverflow.com/questions/3310921/should-i-remove-trailing-slashes-in-meta-tags).

Could you please take a look into it?

 Thx!

**Comment by mlochbaum** [comment]
Used `isSelfClosing` instead of `isVoidElement`...

**Comment by pensierinmusica** [comment]
Thanks!
